changelogs/fragments/ansible-test-docker-ulimit.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-test - Automatically increase the maximum number of open files in a test container from ``1024`` to ``2048``.
```

The default is dependent on the docker daemon, so this is probably more accurate:

```suggestion
  - ansible-test - Change the maximum number of open files in a test container from the default to ``2048``.
```

---

Sounds good, committed, thanks